tests/test_crash_reports.py

```diff
@@ -115,29 +113,22 @@ def test_top_crasher_reports_tab_has_uuid_report(self, mozwebqa):
         reports_table_count = len(crash_signature.reports)
 
         # verify crash reports table is populated
-        Assert.greater(crash_signature.results_count_total, 0)
-        Assert.greater(reports_table_count, 0, "No reports found")
+        assert crash_signature.results_count_total > 0
+        assert reports_table_count > 0, 'No reports found'
 
         most_recent_report = crash_signature.reports[0]
         uuid_report = most_recent_report.click_report_date()
 
         # verify the uuid report page
-        Assert.not_equal(uuid_report.uuid_in_body, "", "UUID not found in body")
-        Assert.not_equal(uuid_report.uuid_in_table, "", "UUID not found in table")
-        Assert.not_equal(uuid_report.signature_in_body, "", "Signature not found in body")
-        Assert.not_equal(uuid_report.signature_in_table, "", "Signature not found in table")
-
-        Assert.equal(uuid_report.uuid_in_body, uuid_report.uuid_in_table,
-                     'UUID in body did not match the UUID in the table: '
-                     'body "%s", table "%s"'
-                     % (uuid_report.uuid_in_body,
-                        uuid_report.uuid_in_table))
-        Assert.contains(uuid_report.signature_in_body,
-                        uuid_report.signature_in_table,
-                        'Signature in body did not match the signature in the '
-                        'table: body "%s", table "%s"'
-                        % (uuid_report.signature_in_body,
-                           uuid_report.signature_in_table))
+        assert uuid_report.uuid_in_body != '', 'UUID not found in body'
+        assert uuid_report.uuid_in_table != '', 'UUID not found in table'
+        assert uuid_report.signature_in_body != '', 'Signature not found in body'
+        assert uuid_report.signature_in_table != '', 'Signature not found in table'
+
+        assert uuid_report.uuid_in_body == uuid_report.uuid_in_table,
+        'UUID in body did not match the UUID in the table'
```

Comment:
I don't think this will work, you probably need the assert in brackets in order to include a newline.